src/utils/biascorr.jl

```diff
@@ -10,74 +55,355 @@ Asymptotic bias correction after fitting binary choice models with a two-/three-
 ## Optional Arguments
 - `L:Int64`: choice of binwidth, see Hahn and Kuersteiner (2011). The default value is 0.
 - `panel_structure`: choose from "classic" or "network". The default value is "classic".
+- `i_symb`
+- `j_symb`
+- `t_symb`
 
-# Notice
-This function only supports binomial distribution and probit/logit link. It also only supports two-way and three-way FE at the moment.
-Always turn on the save option when running `nlreg()` before invoking `BiasCorr()`.
+# Available Model
+We only support the following models:
+- Binomial regression, Logit link, Two-way, Classic
+- Binomial regression, Probit link, Two-way, Classic
+- Binomial regression, Logit link, Two-way, Network
+- Binomial regression, Probit link, Two-way, Network
+- Binomial regression, Logit link, Three-way, Network
+- Binomial regression, Probit link, Three-way, Network
+- Poisson regression, Log link, Three-way, Network
 """
-function BiasCorr(model::GLFixedEffectModel,df::DataFrame;L::Int64=0,panel_structure::Any="classic")
-    # TO-DO: add choice of L (binwidth)
-    # UPDATE: The procedures that use the bandwidth L should be applied to the time dimension. 
-    #         alpaca::biasCorr in R doesn't automatically distinguish between individual fe and time fe. 
-    #         When constructing Bhat (notation inhereted from Fernández-Val and Weidner, Annual Review of Economics (2018), pp120),
-    #         alpaca distinguishes i dimension and t dimension according to the variable positions in the formula,
-    #         but this doesn't show up in their documentation. 
-    #         This can lead to inaccurate results (one can test alpaca::biasCorr with L>0 and with two different formula y~x|i+t and y~x|t+i).
-    #         Is this the best practice? should we also implemented the bandwidth this way? At least we need to include this in the user manual.
-    # TO-DO: check if `df` is sorted. It must be sorted to produce the right result when L > 0
+function BiasCorr(model::GLFixedEffectModel,df::DataFrame;i_symb::Union{Symbol,Nothing}=nothing,j_symb::Union{Symbol,Nothing}=nothing,t_symb::Union{Symbol,Nothing}=nothing,L::Int64=0,panel_structure::Any="classic")
```

Comment:
Let's try to stick to Julia base's [naming conventions](https://docs.julialang.org/en/v1/manual/style-guide/), i.e. lowercase function names and words either together (`biascorr`) or separated by underscores (`bias_correction`). I think I slightly prefer the latter, but don't have strong views.

Also, I'd suggest to make `panel_structure` a `Symbol` and change `"classic"` into `:classic`, `"network"` into `:network`.